code/WorkInProgress/nadir_antenna.dm

```diff
@@ -871,6 +872,24 @@ TYPEINFO(/obj/machinery/transception_pad)
 			M.changeStatus("stunned", 5 SECONDS)
 			M.changeStatus("weakened", 5 SECONDS)
 
+	//if anyone gets stuck inside, eject them. (violently. you got stuck in a prototype teleporter)
+	process()
+		..()
+		if(src.is_transceiving)
```

Comment:
this isn't in the loop and so doesn't do anything

Reply:
Not quite sure what you mean. I double checked after this change and it does still work; the goal of this if/return is to prevent the check from going through if there's already a regular transception in progress at the time of the check.

If there's a way to simply pick one mob from any mobs that are within the objects, that's likely what I would use instead of `for` in this case, but I'm not sure how to make such a thing.

Reply:
That would be `var/mob/M = locate() in src.contents` I think

Reply:
Locateified.